addons/io/org.openhab.io.transport.modbus/src/main/java/org/openhab/io/transport/modbus/WriteTaskImpl.java

```diff
@@ -0,0 +1,63 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.io.transport.modbus;
```

Comment:
move to internal package

Reply:
I wanted it to be public, usable by modbus bindings (also other than the "generic" org.openhab.binding.modbus).

It is an concrete implementation (specialization when writing "registers") for `ModbusWriteRequestBlueprint` interface, which is referenced in the public `ModbusManager` API.

Makes sense?